cogs/polling.py

```diff
@@ -0,0 +1,124 @@
+import discord
+import asyncio
```

Comment:
'asyncio' imported but unused (F401)

To shut the bot up:

```suggestion
#import asyncio
```